crates/codegen/language/definition/src/model/types.rs

```diff
@@ -1,13 +1,15 @@
 pub use self::wrapper::*;
+pub use indexmap::{IndexMap, IndexSet};
```

Comment:
not sure I understand the reason behind that change.

Reply:
I think having each crate declare the types it uses draws a clear boundary, and makes it easier to see what is using what, and helps when grepping/refactoring.
I understand that there is some use cases to crates leaking external types in the ecosystem, but not sure that applies to a monorepo here.
I suggest removing it, and letting whemever needs these types to import/use it themselves.

Reply:
Done in e9add9b49